io/src/main/scala/sbt/io/IOSyntax.scala

```diff
@@ -0,0 +1,13 @@
+package sbt.io
+
+trait IOSyntax extends PathExtra /* with URIExtra */ {
+  type File = java.io.File
+  type URI = java.net.URI
+  type URL = java.net.URL
+
+  def uri(s: String): URI = new URI(s)
+  def file(s: String): File = new File(s)
+  def url(s: String): URL = new URL(s)
+}
+
+object syntax extends IOSyntax
```

Comment:
Indeed it's risky. Much like scalaz providing a scalaz.Scalaz import.

Since this I vaguely remember @retronym saying being too clever in package objects leads to bugs. But that might have been more of an offhand comment in the Scala gitter channel than strong advice to avoid.

It's a tradeoff question. I can be swayed either way.